tommath.h

```diff
@@ -102,8 +102,10 @@ extern "C" {
 /* use arc4random on platforms that support it */
 #ifdef MP_USE_ALT_RAND
     #define MP_GEN_RANDOM()    arc4random()
+    #define MP_GEN_RANDOM_MAX  0xffffffff
```

Comment:
@dod38fr I guess this change would require us to increase `current` of the resulting so, right?

If yes, I'd say it'd be better to move that change to `bn_mp_rand.c` so we only have to increase `revision`